### ISSUE TITLE
Shader render mode crash with Sodium 0.4.9

### ISSUE THREAD

**Issue by RandomTypek** [MeteorDevelopment/meteor-client#3325]
### Describe the bug

Turning on Storage ESP with the shader mode enabled crashes the game. Box mode works fine

[crash-2023-02-08_14.54.29-client.txt](https://github.com/MeteorDevelopment/meteor-client/files/10686414/crash-2023-02-08_14.54.29-client.txt)

### Steps to reproduce

```bash
1. Install Sodium 0.4.9
2. Join world
3. Turn on Storage ESP
```

### Meteor Version

0.5.2-1781

### Minecraft Version

1.19.3

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported (I have searched bug reports on GitHub).

- [X] This is a valid bug (I am able to reproduce this on the latest dev build).

**Comment by ThebestkillerTBK** [comment]
Related to https://github.com/CaffeineMC/sodium-fabric/issues/1602

**Comment by MineGame159** [comment]
Fixed in b4278c0cc512ea92b78861d15b31136bcebd85b0.
